android/build.gradle

```diff
@@ -1,14 +1,11 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.0'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.3.3'
```

Comment:
Yeh kyu hataya com.google.gms? Yeh Firebase ke liye chahiye hota hai

Reply:
See, default code was kotlin+dart, so I had to delete the android directory and create a new java directory, please add the necessary files again.
services JSON, and if any permissions you requested to the android manifest.